packages/mercury/src/components/paginator/_paginator-styles.scss

```diff
@@ -0,0 +1,271 @@
+@use "../../../../common/resets" as *;
+@use "../../base/common" as *;
+@use "../../base/label/label-styles" as *;
+@use "../combo-box/combo-box-styles" as *;
+@use "../input/input-styles" as *;
+
+%paginator {
+  @extend %body-regular-s;
+  display: grid;
+  gap: var(--mer-spacing--xs);
+  grid-template: "items-per-page items-per-page-info . first__button prev__button go-to next__button last__button" 1fr / auto auto 1fr auto auto auto auto;
+
+  &__combo-box {
+    border: 1px solid var(---mer-border-color__primary--focused, #d2ddeb);
+    width: 100px;
```

Comment:
Always use logical properties instead of physical. In this case, `inline-size` is the logical property for `width`.